pkg/commands/process/settings/rules/javascript/react/google_analytics.yml

```diff
@@ -0,0 +1,31 @@
+patterns:
+  - pattern: |
+      ReactGA.event($<DATA_TYPE>)
```

Comment:
Should this be it's own rule? Or should we add this to `third_parties/google_analytics.yml`?

Reply:
It is a non official libarary, so to me it made sense to add it as separate rule.
However, I don't have strong feelings about it and can see your point of it belonging to google analyitics rule.
Am merging it as is in order to move forward but please let me know if you want it changed?